src/applications/search/actions/index.js

```diff
@@ -15,13 +16,30 @@ export function fetchSearchResults(query, page) {
     }
 
     return apiRequest(queryString)
-      .then(response =>
+      .then(response => {
+        if (analyticsMetaInfo) {
+          recordEvent({
+            event: 'view_search_results',
+            'search-page-path': analyticsMetaInfo.path,
```

Comment:
Could we add `?.` in case there's any place we're calling `fetchSearchResults` without providing a 3rd argument? 🙂

Reply:
Oh just saw you wrapped this in an `if` statement for `analyticsMetaInfo`, so disregard my comment if you want! 🤪